### DISCUSSION TITLE
Custom Logo with Docker + nginx-proy not working

### DISCUSSION THREAD

**Discussion by mickymrt** [apache/superset#28206] [Q&A / Help]
Hi
I'm new to superset, trying to customize the log as well in docker deployment with nginx-proxy but having an issue. Followed the steps as per this link https://www.coursebytes.dev/superset/
I have found various other github threads reporting similar issues but none of the suggested solutions work.

I have copied my image file to superset-frontend/src/assets/image
:~/superset$ ll superset-frontend/src/assets/images/ki-nav-brand.png -rw-rw-r-- 1 3571 Apr 18 22:13 superset-frontend/src/assets/images/ki-nav-brand.png 
updated docker/pythonpath_dev/superset_config_docker.py to include
APP_ICON = "/static/assets/images/ki-nav-brand.png"
restarted superset, but he image is broken
This is very frustrating!
Even tried to replace the superset-logo-horiz.png with ki-nav-brand.png, but the original superset logo is displayed instead of my customized one, which is odd...
APP_NAME = and LOGO_TOOLTIP = settings take affect ok

I suspect that the issue might be to do with nginx-proxy. I can see this error when starting superset 
 I see these errors when loading

nginx-proxy-1         | nginx.1     | <examplename>.org 10.6.120.143 - - [24/Apr/2024:23:34:21 +0000] "GET /static/appbuilder/css/fontawesome/solid.min.css HTTP/2.0" 200 319 "https://<examplename>.org/login/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36" "172.18.0.6:8088" superset_app          | 172.18.0.4 - - [24/Apr/2024:23:34:21 +0000] "GET /static/appbuilder/css/ab.css HTTP/1.1" 200 294 "https://<examplename>.org/login/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36" superset_app          | 172.18.0.4 - - [24/Apr/2024:23:34:21 +0000] "GET /static/appbuilder/css/fontawesome/solid.min.css HTTP/1.1" 200 319 "https://<examplename>.org/login/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36" superset_app          | HTTPException superset_app          | Traceback (most recent call last): superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request superset_app          |     rv = self.dispatch_request() superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1788, in dispatch_request superset_app          |     self.raise_routing_exception(req) superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1770, in raise_routing_exception superset_app          |     raise request.routing_exception  # type: ignore superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/ctx.py", line 351, in match_request superset_app          |     result = self.url_adapter.match(return_rule=True)  # type: ignore superset_app          |   File "/usr/local/lib/python3.10/site-packages/werkzeug/routing/map.py", line 652, in match superset_app          |     raise NotFound() from None superset_app          | werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. superset_app          | 2024-04-24 23:34:21,140:WARNING:superset.views.base:HTTPException superset_app          | Traceback (most recent call last): superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request superset_app          |     rv = self.dispatch_request() superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1788, in dispatch_request superset_app          |     self.raise_routing_exception(req) superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1770, in raise_routing_exception superset_app          |     raise request.routing_exception  # type: ignore superset_app          |   File "/usr/local/lib/python3.10/site-packages/flask/ctx.py", line 351, in match_request superset_app          |     result = self.url_adapter.match(return_rule=True)  # type: ignore superset_app          |   File "/usr/local/lib/python3.10/site-packages/werkzeug/routing/map.py", line 652, in match superset_app          |     raise NotFound() from None superset_app          | werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. 
please assist!
![2024-04-25_9-34-35](https://github.com/apache/superset/assets/168042487/56092b9a-8dda-4721-a0c9-1bc2e3d7209b)

**Comment by MfgIsiaka** [comment]
> Hi I'm new to superset, trying to customize the log as well in docker deployment with nginx-proxy but having an issue. Followed the steps as per this link https://www.coursebytes.dev/superset/ I have found various other github threads reporting similar issues but none of the suggested solutions work.
> 
> I have copied my image file to superset-frontend/src/assets/image :~/superset$ ll superset-frontend/src/assets/images/ki-nav-brand.png -rw-rw-r-- 1 3571 Apr 18 22:13 superset-frontend/src/assets/images/ki-nav-brand.png updated docker/pythonpath_dev/superset_config_docker.py to include APP_ICON = "/static/assets/images/ki-nav-brand.png" restarted superset, but he image is broken This is very frustrating! Even tried to replace the superset-logo-horiz.png with ki-nav-brand.png, but the original superset logo is displayed instead of my customized one, which is odd... APP_NAME = and LOGO_TOOLTIP = settings take affect ok
> 
> I suspect that the issue might be to do with nginx-proxy. I can see this error when starting superset I see these errors when loading
> 
> nginx-proxy-1 | nginx.1 | .org 10.6.120.143 - - [24/Apr/2024:23:34:21 +0000] "GET /static/appbuilder/css/fontawesome/solid.min.css HTTP/2.0" 200 319 "https://.org/login/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36" "172.18.0.6:8088" superset_app | 172.18.0.4 - - [24/Apr/2024:23:34:21 +0000] "GET /static/appbuilder/css/ab.css HTTP/1.1" 200 294 "https://.org/login/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36" superset_app | 172.18.0.4 - - [24/Apr/2024:23:34:21 +0000] "GET /static/appbuilder/css/fontawesome/solid.min.css HTTP/1.1" 200 319 "https://.org/login/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36" superset_app | HTTPException superset_app | Traceback (most recent call last): superset_app | File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request superset_app | rv = self.dispatch_request() superset_app | File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1788, in dispatch_request superset_app | self.raise_routing_exception(req) superset_app | File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1770, in raise_routing_exception superset_app | raise request.routing_exception # type: ignore superset_app | File "/usr/local/lib/python3.10/site-packages/flask/ctx.py", line 351, in match_request superset_app | result = self.url_adapter.match(return_rule=True) # type: ignore superset_app | File "/usr/local/lib/python3.10/site-packages/werkzeug/routing/map.py", line 652, in match superset_app | raise NotFound() from None superset_app | werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. superset_app | 2024-04-24 23:34:21,140:WARNING:superset.views.base:HTTPException superset_app | Traceback (most recent call last): superset_app | File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1823, in full_dispatch_request superset_app | rv = self.dispatch_request() superset_app | File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1788, in dispatch_request superset_app | self.raise_routing_exception(req) superset_app | File "/usr/local/lib/python3.10/site-packages/flask/app.py", line 1770, in raise_routing_exception superset_app | raise request.routing_exception # type: ignore superset_app | File "/usr/local/lib/python3.10/site-packages/flask/ctx.py", line 351, in match_request superset_app | result = self.url_adapter.match(return_rule=True) # type: ignore superset_app | File "/usr/local/lib/python3.10/site-packages/werkzeug/routing/map.py", line 652, in match superset_app | raise NotFound() from None superset_app | werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again. please assist! ![2024-04-25_9-34-35](https://private-user-images.githubusercontent.com/168042487/325422978-56092b9a-8dda-4721-a0c9-1bc2e3d7209b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzA3NDM2ODcsIm5iZiI6MTc3MDc0MzM4NywicGF0aCI6Ii8xNjgwNDI0ODcvMzI1NDIyOTc4LTU2MDkyYjlhLThkZGEtNDcyMS1hMGM5LTFiYzJlM2Q3MjA5Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDIxMFQxNzA5NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNDIyYzAwNWNjOTYxYWI4YmYwYTY4YWRhMjM4NWUxNzk2NTVmN2U5Mzk0MWUxYjhmMWIyY2RiMTg5NTZlZDhiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Xg7PcayLpp4ZAqUyF8UafY9RRtyZVcUYjDpMmQfJJfQ)

Hey did you find solution to this problem?
